WordPress/src/main/res/layout/site_creation_intents_fragment.xml

```diff
@@ -57,4 +57,13 @@
 
     </com.google.android.material.appbar.AppBarLayout>
 
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycler_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:importantForAccessibility="no"
+        android:scrollbars="vertical"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior"
```

Comment:
What do you think about adding `tools:listitem="@layout/site_creation_intents_item"` for the preview during editing?

<details>
<summary>Preview</summary>

<img width="343" alt="Screenshot 2022-03-21 at 17 15 44" src="https://user-images.githubusercontent.com/4588074/159304014-025069ab-67e4-4135-82ad-1a1283a0ad9f.png">
</details>

Reply:
Perfect! I was actually looking for that, but my autocomplete was acting funky at the time, and I didn't find it. I similarly added the tools:background colors (especially useful for dark mode styles :smile: ).